after setting device with ```make dev ``` i run ```make monitor``` and it fails:
```
make[1]: Nothing to be done for 'check'.
docker run \
        --rm \
        -ti \
         \
        -v pio-home:/home/developer:rw \
        -v /home/ikstream/docs/workspace/esp32-core/docker/..:/mnt:rw \
        -w /mnt \
        --hostname platformio \
         \
        lifesensor/platformio:4.1.0 \
        /bin/sh -c 'sudo chgrp developer /dev/ttyUSB1;          python3                         /home/developer/.platformio/packages/framework-espidf/tools/idf_monitor.py                     /mnt/.pio/build/pico32/firmware.elf                      --port /dev/ttyUSB1'
chgrp: cannot access '/dev/ttyUSB1': No such file or directory
--- idf_monitor on /dev/ttyUSB1 115200 ---
--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H ---

Exception in thread Thread-2:
Traceback (most recent call last):
  File "/home/developer/.local/lib/python3.6/site-packages/serial/serialposix.py", line 265, in open
    self.fd = os.open(self.portstr, os.O_RDWR | os.O_NOCTTY | os.O_NONBLOCK)
FileNotFoundError: [Errno 2] No such file or directory: '/dev/ttyUSB1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/home/developer/.platformio/packages/framework-espidf/tools/idf_monitor.py", line 133, in _run_outer
    self.run()
  File "/home/developer/.platformio/packages/framework-espidf/tools/idf_monitor.py", line 222, in run
    self.serial.open()
  File "/home/developer/.local/lib/python3.6/site-packages/serial/serialposix.py", line 268, in open
    raise SerialException(msg.errno, "could not open port {}: {}".format(self._port, msg))
serial.serialutil.SerialException: [Errno 2] could not open port /dev/ttyUSB1: [Errno 2] No such file or directory: '/dev/ttyUSB1'
```

The device seems to be not passed to the ```make monitor``` target correctly

--- a/Makefile
+++ b/Makefile
@@ -86,7 +86,7 @@ distclean-build: clean-build
 ### flash targets ###
 
 TARGET += flash
-HELP_flash = flashes project to esp
+HELP_flash = flashes project to esp. Use DEV=path to provide  path to device or use make dev
 .PHONY: flash
 flash: | check-flash
 	@make --no-print-directory -C docker \

--- a/Makefile
+++ b/Makefile
@@ -107,7 +107,7 @@ distclean-flash: clean-flash
 ### monitor targets ###
 
 TARGET += monitor
-HELP_monitor = connects to esp32 via serial
+HELP_monitor = connects to esp32 via serial. Use DEV=path to provide path to device or use make dev
 .PHONY: monitor
 monitor: | check-monitor
 	@make --no-print-directory -C docker \

--- a/Makefile
+++ b/Makefile
@@ -144,6 +144,7 @@ dev:
 		--menu "choose which device to flash" 0 0 0 \
 		--file /dev/stdin \
 	>> ./.dev
+	clear
 
 TARGET_dev += check-dev
 HELP_check-dev = checks if device is specified

